web/src/features/charts/CarbonChart.tsx

```diff
@@ -31,7 +32,7 @@ function CarbonChart({ datetimes, timeAverage }: CarbonChartProps) {
     return <NotEnoughDataMessage title="country-history.carbonintensity" />;
   }
   return (
-    <>
+    <GraphCard className="pb-2">
```

Comment:
In the Figma the unit have moved places from the right side to under the title. Do we want to do that in this PR or do you want to make a follow up PR for that?

Reply:
I'll do it in another PR because I'm not completely sure of what we want to write as the units

Reply:
The vertical axis cuts into the graph now. Which I believe is due to the margin the unit is causing.
<img width="461" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/215e505d-be3d-416a-bd1f-1a257f403863">

As for what to write for the units I think that should be the same as the unit's that are already there for now.

Reply:
True, I hope it is fixed now:)